releasenotes/notes/add-v4-span-links-field-moon-331e24faf41ec6f4.yaml

```diff
@@ -0,0 +1,6 @@
+
+features:
+  - |
+    Add `span_links` field to  trace payload. This field
+    contains a list of casual relationships between spans and is only
+    populated when v0.4 trace api is used.
```

Comment:
In general, the audience is a Datadog customer that has no context about how the tracer or the Agent work. e.g. why do they care about a `span_links` field?

What you probably really want to say is that this adds support for span links in the agent, link to some docs, mention that the feature is in beta. etc.